support-frontend/assets/components/subscriptionCheckouts/personalDetails.tsx

```diff
@@ -87,7 +87,7 @@ export default function PersonalDetails(props: PropTypes): JSX.Element {
 		e: React.FocusEvent<HTMLInputElement>,
 	) => {
 		if (props.fetchAndStoreUserType) {
-			props.fetchAndStoreUserType(e.target.value);
+			void props.fetchAndStoreUserType(e.target.value);
```

Comment:
[From the docs](https://www.typescriptlang.org/docs/handbook/2/functions.html#return-type-void)

> function type with a void return type (type voidFunc = () => void), when implemented, can return any other value, but it will be ignored.

Which is similar to `unknown` in the sense that it can't really be used but also `undefined` as in it will not be used.

In this instance it's to indicate "This value is not meaningful and will not be used").